client/src/entrypoints/admin/comments.js

```diff
@@ -11,10 +13,15 @@ window.comments = (() => {
 
   /**
    * Returns true if the provided keyboard event is using the 'add/focus comment' keyboard
-   * shortcut
+   * shortcut and custom keyboard shortcuts enabled by user in settings
```

Comment:
```suggestion
   * shortcut and custom keyboard shortcuts enabled by user in settings.
```

Just a small nitpick.